script/setup/install-cni

```diff
@@ -20,8 +20,7 @@
 # The commit defined in go.mod
 #
 set -eu -o pipefail
-
-CNI_COMMIT=${1:-$(grep containernetworking/plugins "$GOPATH"/src/github.com/containerd/containerd/go.mod | awk '{print $2}')}
+CNI_COMMIT=${1:-$(go list -f "{{.Version}}" -m github.com/containernetworking/plugins)}
```

Comment:
Minor nit: can you keep the empty line? (reduces code-churn, and looks like we have an empty line after everywhere else)

Reply:
I remember, thanks!